documentation/configuration.md

```diff
@@ -849,6 +849,84 @@ In addition to enabling custom providers, `dotnet monitor` also allows you to di
 
 The Queue Message's payload will be the blob name (`<BlobPrefix>/<ArtifactName>`; using the above example with an artifact named `mydump.dmp`, this would be `artifacts/mydump.dmp`) that is being egressed to blob storage. This is designed to be easily integrated into an Azure Function that triggers whenever a new message is added to the queue, providing you with the contents of the artifact as a stream. See [Azure Blob storage input binding for Azure Functions](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-blob-input?tabs=csharp#example) for an example.
 
+### S3 storage egress provider
+
+| Name | Type | Required | Description |
+|---|---|---|---|
+| endpoint | string | true | An optional endpoint of S3 storage service. Can be left empty in case of using AWS. |
+| bucketName | string | true | The name of the s3 Bucket to which the blob will be egressed |
```

Comment:
There are several cases where `s3` is used (note the lowercase s) where it should always be `S3` (note the capital S). Please fix to be capital on all cases.